dbus_next/message_bus.py

```diff
@@ -633,6 +640,19 @@ def reply_handler(reply, err):
                     interface='org.freedesktop.DBus.Peer',
                     member='GetMachineId'), reply_handler)
 
+    def _default_object_manager(self, msg):
+        result = {}
+
+        for node in self._path_exports:
+            if not node.startswith(msg.path):
```

Comment:
From the spec:

> their object paths start with the ObjectManager's object path plus '/'.

Reply:
Yes, I wasn't sure about this.

So as it is at the moment, having interfaces like 
> `/com/example/sample0`
`/com/example/sample0/ex0`

ObjectManager on sample0 returns: 
>`{'/com/example/sample0': {`
    `'com.example.SampleInterface0':  {'Bar': 105, 'Foo': 42},`
    `'com.example.SampleInterface1':  {'Bar': 105, 'Foo': 42}},`
`'/com/example/sample0/ex0': {`
    `'com.example.OtherInterface0':  {'Bar': 105,  'Foo': 42}}}`

while that on ex0:
>`{'/com/example/sample0/ex0': {`
    `'com.example.OtherInterface0':  {'Bar': 105,'Foo': 42}}}`


Is it then suppose to be:

>`{'/com/example/sample0/ex0': {`
    `'com.example.OtherInterface0':  {'Bar': 105,'Foo': 42}}}`
 
>`{}`

?

I tried to find some real life examples but not many programs implement this interface and each seam to do their own thing. 
Alternatively I cold also just restrict ObjectManager to be available only on `'/'` however I don't think that's according to spec either.

Reply:
> Is it then suppose to be:

Yeah the spec says it's that second thing how I read it.

Reply:
I changed the behavior to work like in spec and added test in test_standard_interfaces